features/FEATURE_BLE/source/generic/GenericSecurityManager.cpp

```diff
@@ -370,7 +380,7 @@ class GenericSecurityManager : public SecurityManager,
     }
 
     virtual ble_error_t passkeyEntered(connection_handle_t connection, Passkey_t passkey) {
-        return pal.passkey_request_reply(connection, passkey);
+        return pal.passkey_request_reply(connection, PasskeyAsci::to_num(passkey));
```

Comment:
no, that passkey is the asci version

Reply:
according to the compiler and the code it is the num version:  https://github.com/paul-szczepanek-arm/mbed-os/blob/palsm/features/FEATURE_BLE/ble/pal/PalSecurityManager.h#L458

Reply:
sorry, I was thinking of the    passkeyEntered(Gap::Handle_t handle, Passkey_t passkey)

Reply:
no, that is right

Reply:
I run compilation to ensure that typewise it's fine. That's why I made this PR.